.github/workflows/main.yaml

```diff
@@ -0,0 +1,34 @@
+name: Test yugabyte-db-action
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
We should fire the repo dispatch action from the main repo and test each time a new version comes out.

Reply:
Okay. I'll update the required changes and create the PR for the `yugabyte-db` repo as well for repository dispatch events.